src/tests/ut/receptionist/shift_test.hpp

```diff
@@ -1,7 +1,11 @@
 #pragma once
-#include "../../../app/receptionist/shift.hpp"
+
+#include "shift.hpp"
 #include "gtest/gtest.h"
 
+namespace unit_test::receptionist
```

Comment:
remove in hpp, only tests should be in anonymous namespace

Reply:
done